HelloWorld/HelloWorld/Program.cs

```diff
@@ -6,7 +6,7 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            Console.WriteLine("Hello World! And Good luck with LEAP!");
```

Comment:
If we could screenshot things we should add this to the motivation slack channel :) LGTM